src/Main.java

```diff
@@ -0,0 +1,23 @@
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+
+        int workTime;
+
+        System.out.println("Gib eine Zahl zwischen 0 & 24 ein");
+        workTime = sc.nextInt();

```

Comment:
anstelle Deklarierung und Initialisierung zu splitten würd ich das in ein statement zusammenfassen - Variablen sollen so nah wie möglich an der ersten Verwendung angelegt werden, dass erhöht die lesbarkeit